src/pages/docs/liveobjects/rest-api-usage.mdx

```diff
@@ -67,6 +67,8 @@ The response includes the IDs of the objects on the channel:
 ```
 </Code>
 
+This request is billed per object ID returned - each object ID counts as one billable message.
```

Comment:
If we're no longer really highlighting object IDs as a primary concept in the API, perhaps this could just say

> This request is billed per the number of objects returned - each distinct object counts as one billable message